docs/cli.md

```diff
@@ -4,21 +4,149 @@ title: babel-cli
 sidebar_label: babel-cli
 ---
 
-In addition, various entry point scripts live in the top-level package at `@babel/cli/bin`.
+Babel comes with a built-in CLI which can be used to compile files from the command line.
 
-There is a shell-executable utility script, `babel-external-helpers.js`, and the main Babel cli script, `babel.js`.
+In addition, various entry point scripts live in the top-level package at `@babel/cli/bin`. There is a shell-executable utility script, `babel-external-helpers.js`, and the main Babel cli script, `babel.js`.
 
 ## Install
 
+While you _can_ install Babel CLI globally on your machine, it's much better
+to install it **locally** project by project.
+
+There are two primary reasons for this.
+
+1. Different projects on the same machine can depend on different versions of
+     Babel allowing you to update one at a time.
+2. It means you do not have an implicit dependency on the environment you are
+     working in. Making your project far more portable and easier to setup.
+
+We can install Babel CLI locally by running:
+
 ```sh
 npm install --save-dev @babel/core @babel/cli
 ```
 
-## Usage
+> **Note:** If you do not have a `package.json`, create one before installing. This will ensure proper interaction with the `npx` command.
+
+After that finishes installing, your `package.json` file should include:
+
+```diff
+{
+  "devDependencies": {
++   "@babel/cli": "^7.0.0",
++   "@babel/core": "^7.0.0"
+  }
+}
+```
+
+## Basic Usage 
 
 ```sh
 babel script.js
 ```
 
-For more in depth documentation see: http://babeljs.io/docs/usage/cli/
+## babel
```

Comment:
With babel-node being a separate page this heading is probably not needed anymore.